package.json

```
@@ -82,6 +82,21 @@
       "classnames": "<rootDir>/node_modules/classnames/index.js"
     }
   },
+  "nyc": {
+    "report-dir": "cypress-coverage",
+    "include": [
+      "src/Mutations",
+      "src/PresentationalComponents",
+      "src/Modules",
+      "src/SmartComponents",
+      "src/__factories__",
+      "src/Utilities"
```

Is there a reason to omit other files under `src`? Can we use just:
```suggestion
      "src/**/*"
```